metallb/deploy_operator.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/bash
+
+METALLB_OPERATOR_REPO=${METALLB_OPERATOR_REPO:-"https://github.com/metallb/metallb-operator.git"}
```

Comment:
shouldn't we point to the openshift repo?

Reply:
i wanted to stick to the e2e tests default repo (the u/s one) here:
https://github.com/openshift-metal3/dev-scripts/blob/eeb48a787cc2fca426babdb2eb5e80779e9ce945/metallb/run_e2e.sh#L3
so i thought to set the repo to the openshift one in the CI config here:
https://github.com/openshift/release/blob/928719efeaa06d43c59046bd0aca434a283bfd18/ci-operator/config/openshift/metallb/openshift-metallb-main.yaml#L60

Reply:
I'd explicitly pin to a specific commit (either here or under release)